object-cache.php

```diff
@@ -712,6 +712,7 @@ protected function _connect_redis() {
 
 		$this->redis = new Redis;
 		$port = ! empty( $redis_server['port'] ) ? $redis_server['port'] : 6379;
+		if(file_exists($redis_server['host']) && (filetype( $redis_server['host'])=='socket')) $port = null;
```

Comment:
Can you follow WP Coding Standards?

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style